src/pages/Add-items/Add-items.js

```diff
@@ -76,82 +79,95 @@ export const AddItems = () => {
+            <div className="filter-item inputField">
               <input
-                id="notSoon"
-                type="radio"
-                name="howSoon"
                 required
-                value="30"
-                checked={product.howSoon === '30'}
+                id="name"
+                className="filter-input"
+                type="text"
+                name="name"
+                value={product.name}
                 onChange={handleChangeProduct}
+                ref={inputRef}
+                placeholder="New product"
               />
-              Not soon
-            </label>
-          </div>
-        </fieldset>
+            </div>
+          </label>
+
+          <p className="how-soon">How soon will you buy this again?</p>
+          <fieldset>
+            <div>
+              <label htmlFor="soon">
+                <input
+                  id="soon"
+                  type="radio"
+                  name="howSoon"
+                  required
+                  value="7"
+                  checked={product.howSoon === '7'}
+                  onChange={handleChangeProduct}
+                />
+                Weekly
+              </label>
+            </div>
+            <div>
+              <label htmlFor="kindOfSoon">
+                <input
+                  id="kindOfSoon"
+                  type="radio"
+                  name="howSoon"
+                  required
+                  value="14"
+                  checked={product.howSoon === '14'}
+                  onChange={handleChangeProduct}
+                />
+                Biweekly
+              </label>
+            </div>
+            <div>
+              <label htmlFor="notSoon">
+                <input
+                  id="notSoon"
+                  type="radio"
+                  name="howSoon"
+                  required
+                  value="30"
+                  checked={product.howSoon === '30'}
+                  onChange={handleChangeProduct}
+                />
+                Monthly
+              </label>
+            </div>
+          </fieldset>
+        </div>
         <div className="button-container">
-          <button type="submit">Submit</button>
+          <button className="button-container" type="submit">
+            Submit
+          </button>
         </div>
       </form>
+
       <Modal
-        children={'This product is duplicated'}
+        children={'Your product already exists'}
         modalClass={modalDuplicatedProductMsg.modalClass}
         handleClose={modalDuplicatedProductMsg.hideModal}
+        iconMaterial={'error'}
+        colorIcon={'md-light_error'}
       />
       <Modal
         children={'Your product was successfully added'}
         modalClass={modalProductAdded.modalClass}
         handleClose={modalProductAdded.hideModal}
-      />
+        iconMaterial={'check_circle'}
+        colorIcon={'md-light_success'}
+      ></Modal>
+
       <Nav />
-    </>
+    </main>
```

Comment:
Sugerencia. No blocking.

Personalmente, sugeriría mover de posición el componente `Nav`, fuera del `main`. Porque no es parte necesariamente del contenido principal, sino algo externo (pueden usar React fragments `<>content</>`).

```suggestion
</>
... more code...
    </main>
    <Nav />
 </>
```

Además que ayudará con un detalle con la versión móvil.

Reply:
Sería genial, aplicarlo a las demás vistas, como el de listado de productos. 😄 🙏

Reply:
***c6231c109871a2ca93e66bfa807f4bb0e7f23644*** listo!